systest/Makefile

```diff
@@ -138,9 +146,37 @@ cleanup_failed_tlc:
 
 # BRANCH MAKE TARGETS:
 #
+# These make targets invoke OTHER targets defined in this file.  They are not
+# used to invoke particular tests, rather they manage test infrastructure, and
+# conform to the standard interface expected by buildbot.
+#
+# Run all tests, this is what buildbot calls: make functest
+# These are all of the functests that we want to run as part of bbot job
+# We ignore the return value so that we run all of the tests
+functest: install_tlc install_test_infra
+	-$(MAKE) -C . tempest_12.1.1_overcloud
+	-$(MAKE) -C . tempest_13.0.0_overcloud
+	-$(MAKE) -C . tempest_12.1.1_undercloud
+	-$(MAKE) -C . tempest_12.1.1_undercloud_gre
+	-$(MAKE) -C . tempest_12.1.1_undercloud_vlan
+	-$(MAKE) -C . tempest_13.0.0_undercloud
+	-$(MAKE) -C . tempest_13.0.0_undercloud_gre
+	-$(MAKE) -C . tempest_13.0.0_undercloud_vlan
+
```

Comment:
what pattern should I use, then? I *thought* I was simply to repeat patterns that exist...